includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -552,6 +552,8 @@ public function sanitize_settings( $settings ) {
 	 *
 	 * @return bool
 	 * @since  1.2
+	 *
+	 * @todo Is this function supposed to be here?
```

Comment:
This function doesn't seem like it should be here -- it exists in another class, and this copy isn't used anywhere -- but I left it alone.

Reply:
This is most likely a leftover from the refactoring effort. I'd imagine we'd be safe to remove it but should test that in a separate PR.